transformations/spellchecker_perturbation/transformation.py

```diff
@@ -0,0 +1,70 @@
+import random
+
+import enchant
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def is_case_same(char_1: str, char_2: str) -> bool:
```

Comment:
Is there a need for this function to be outside the class? Is it used anywhere else?

Reply:
No, will change this as requested.